src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -152,10 +153,11 @@ protected void updateItem(final TableRow item, final boolean empty) {
                      setText(null);
                   } else {
                      final String notes = item.getNotes();
-                     final Label label = new Label(notes.isEmpty() ? EMPTY_NOTE : notes);
-                     label.setUnderline(item.isUnderlined());
-                     label.setTooltip(new Tooltip(notes));
-                     this.setGraphic(label);
+                     final Text text = new Text(notes.isEmpty() ? EMPTY_NOTE : notes);
+                     this.setText(text.getText());
+
+                     final Circle circle = new Circle(6, item.getProjectColor());
```

Comment:
this is not right or? we can only set a graphic if we also have a color? Creating a color with "null" as color (which happens for the notes) is not okay.
if we don't have a color we need to set graphic to 'null'.

Reply:
Graphic no only get set if the  item.getProjectColor is not null